cpp/include/Ice/Proxy.h

```diff
@@ -138,7 +152,7 @@ class InvokeOutgoingAsyncT : public OutgoingAsync
             const ::Ice::Byte* encaps;
             ::Ice::Int sz;
             stream->readEncapsulation(encaps, sz);
-            return R { ok, { encaps, encaps + sz } };
+            return makeIceInvokeResult<R>(ok, encaps, sz);
```

Comment:
I didn't add a test for this but seems it will be also bogus when `R` is `Ice_invokeResult`.

Reply:
```
using Outgoing = ::IceInternal::InvokeLambdaOutgoing<::Ice::Object::Ice_invokeResult>;
::std::function<void(::Ice::Object::Ice_invokeResult&&)> r;
```

And

```
template<typename R>
class InvokeLambdaOutgoing : public InvokeOutgoingAsyncT<R>, public LambdaInvoke
```

Reply:
Ok.